command/genesis/params.go

```diff
@@ -197,21 +232,30 @@ func (p *genesisParams) initValidatorSet() error {
 }
 
 func (p *genesisParams) isValidatorNumberValid() bool {
-	return uint64(len(p.ibftValidators)) <= p.maxNumValidators
+	return p.ibftValidators == nil || uint64(p.ibftValidators.Len()) <= p.maxNumValidators
```

Comment:
How is it correct for this check to return ✅ if the `p.ibftValidators` is `nil`?